site/src/docs/getting-started/tutorials/migrate-to-4.0.0.mdx

```diff
@@ -195,75 +195,95 @@ There are updates to lineheights to improve readability especially on Headings.
 #### Black-60 changed to more accesible shade
 Design token black-60 is changed from #666666 to #595959 to improve contrast with other colours. With only bus-dark and black-90 the contrast gets worse, all other colours the contrast improves. If needed, verify the used colours.
 
+---
+
 ## Design kit
```

Comment:
Not related to this line but since I started the review I'll add it here: 

I was thinking if we could have the 4.0.0 Migration guide on top of the accordion links since it's the most recent and relevant. (needs to be changed in gatsby not here?)

**EDIT note**: or is the order always alphabetical?

<img width="233" alt="Screenshot 2024-11-06 at 11 16 26" src="https://github.com/user-attachments/assets/114f18c6-cc08-49fe-b716-e8142c5111d2">